core/templates/components/shared-component.module.ts

```diff
@@ -196,16 +202,18 @@ const firebaseAuthProviders = AuthService.firebaseAuthIsEnabled ? [
     // Components, directives, and pipes.
     AudioFileUploaderComponent,
     BackgroundBannerComponent,
+    CreateNewSkillModalComponent,
     CreateActivityButtonComponent,
     CreateActivityModalComponent,
     ExplorationSummaryTileComponent,
     CollectionSummaryTileComponent,
     LazyLoadingComponent,
-    FilterForMatchingSubstringPipe,
+    FocusOnDirective,
```

Comment:
Why is this here? Can you check if this is exported as a part of `DirectivesModule`.

Reply:
fixed.